core-bundle/tests/Security/TwoFactor/AuthenticatorTest.php

```diff
@@ -126,4 +126,16 @@ public function testGeneratesTheQrCode(): void
         $this->assertSame(5897, \strlen($svg));
         $this->assertSame(0, strpos($svg, $beginSvg));
     }
+
+    private function randomBytes(int $bytesCount, int $seed): string
```

Comment:
I guess you should also rename this method.

Reply:
Yes, need to integrate changes from #2879

Reply:
Done in 5271027181ae7f966ebd1e450fa45f3eb5cceba8